### PR TITLE
Add Azure Maps 101 quickstart

### PR DESCRIPTION
**Pull request by philmea**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@philmea - check this PR for updates that may be needed to documentation that references this sample.

**Comment by philmea** [comment]
@bmoore-msft I will be publishing a corresponding article to support this example.

File: 101-maps-create/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Create an Azure Maps
```
Comment (bmoore-msft) [inline 101-maps-create/README.md:1]: Create an Azure Maps Account ?

File: 101-maps-create/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-maps-create%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-maps-create%2Fazuredeploy.json)
+
+This template creates an Azure Maps account and lists the primary key as output. To learn more about how to deploy the template, see the [article](https://docs.microsoft.com/azure/azure-maps/how-to-create-template) article.
```
Comment (bmoore-msft) [inline 101-maps-create/README.md:16]: update this when you remove the output

File: 101-maps-create/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name":{
```
Comment (bmoore-msft) [inline 101-maps-create/azuredeploy.json:5]: use camel case in param names, e.g. "accountName"

File: 101-maps-create/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            "type": "string",
+            "defaultValue": "exampleMap",
```
Comment (bmoore-msft) [inline 101-maps-create/azuredeploy.json:7]: does this need to be globally unique?  If so, use uniqueString and update the description

File: 101-maps-create/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                "S0",
+                "S1"
+            ],
+            "defaultValue": "S0",
+            "metadata": {
+                "description": "The pricing tier for the account. Use S0 for small-scale development. Use S1 for large-scale applications."
+            }
+        }
+    },
+    "variables": {
```
Comment (bmoore-msft) [inline 101-maps-create/azuredeploy.json:24]: remove this property

File: 101-maps-create/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+            "location": "global",
+            "tags": {
+            },
+            "sku": {
+                "name": "[parameters('pricingTier')]"
+            }
+        }
+    ],
+    "outputs": {
+        "primaryKey": {
```
Comment (bmoore-msft) [inline 101-maps-create/azuredeploy.json:40]: remove this - you're not allowed to output secrets.

File: 101-maps-create/azuredeploy.parameters.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name": {
```
Comment (bmoore-msft) [inline 101-maps-create/azuredeploy.parameters.json:5]: remove these since you have defaultValues in the template

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-maps-create/azuredeploy.json (hunk 6 above)
Comment (philmea) [reply 101-maps-create/azuredeploy.json:40]: This is used to show that the key can be captured and used in additional deployment steps. Can there be an exception?

File: 101-maps-create/azuredeploy.parameters.json (hunk 7 above)
Comment (philmea) [reply 101-maps-create/azuredeploy.parameters.json:5]: Remove the parameters file entirely?

File: 101-maps-create/azuredeploy.json (hunk 6 above)
Comment (bmoore-msft) [reply 101-maps-create/azuredeploy.json:40]: No, unfortunately (or maybe it's a good thing) customers tend to use these examples in production environments and adopt the bad practice.  If there is a use case for consuming the key, you can always add that to the sample or another one.

File: 101-maps-create/azuredeploy.parameters.json (hunk 7 above)
Comment (bmoore-msft) [reply 101-maps-create/azuredeploy.parameters.json:5]: no, just empty the parameters property, e.g. 
```json
{
  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
  "contentVersion": "1.0.0.0",
  "parameters": { }
}
```

**Comment by bmoore-msft** [comment]
@philmea - check this PR for updates that may be needed to documentation that references this sample.

**Pull request merged**